lib/blazer/engine.rb

```diff
@@ -3,9 +3,14 @@ class Engine < ::Rails::Engine
     isolate_namespace Blazer
 
     initializer "blazer" do |app|
+      ActiveSupport.on_load :action_view do
+        include Blazer::BaseHelper
```

Comment:
This is required so we can call `blazer_js_var`.
I originally has this in my app, but I didn't like the feeling of this extra step. I wanted to get to a simple API of just a render call + activerecord call.

I don't think this is a concern, especially as we have prefixed all methods with `blazer_`.

Can anyone think of trouble this could cause?